src/index.html

```diff
@@ -1,13 +1,389 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>MET landing</title>
+    <link
+    rel="icon"
+    type="image/x-icon"
+    href="/images/favicon.ico"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+    rel="preconnect"
+    href="https://fonts.gstatic.com"
+    crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__body">
+
+    <header class="header">
+
+      <div class="top-bar">
+        <a
+        href="#"
+        class="top-bar__logo"
+        alt="logo"
+        >
+          <img
+          src="./images/logo.svg"
+          class="logo"
+          alt="Logo icon"
+          >
```

Comment:
```suggestion
          href="#"
          class="top-bar__logo"
          alt="logo"
        >
          <img
            src="./images/logo.svg"
            class="logo"
            alt="Logo icon"
          >
```
Consider fixing indents according to the code style guide